test/client/domains.js

```diff
@@ -38,4 +44,47 @@ describe(`domains test`, () => {
             }
         }
     });
+
+    it('getPayPalLoggerDomain should return the logger domain when is a local environment', () => {
+        window.__ENV__ = ENV.LOCAL;
+        const result = getPayPalLoggerDomain();
+
+        if (result !== 'https://mock://msmaster.qa.paypal.com') {
+            throw new Error(`should get the logger domain "https://mock://msmaster.qa.paypal.com", but got: ${ result }`);
+        }
+    });
+
+    it('getPayPalLoggerDomain should thrown an Error when is a local environment and the stage host is undefined', () => {
+        const previousEnv =  window.__ENV__;
+        const previousStageHost =  window.__STAGE_HOST__;
+        window.__ENV__ = ENV.LOCAL;
+        window.__STAGE_HOST__ = undefined;
+        try {
+            getPayPalLoggerDomain();
+        } catch (err) {
+            if (err.message !== 'No stage host found') {
+                throw new Error(`should thrown exception with message "No stage host found", but got ${ err.message }`);
+            }
+        }
+        window.__ENV__ = previousEnv;
+        window.__STAGE_HOST__ = previousStageHost;
```

Comment:
Can you reset this `window.__STAGE_HOST__` in the beforeEach() as well?
```suggestion
```

Reply:
Not sure why we should move the cleanup to the beforeEach cycle, we modify the `__STAGE_HOST__ ` only for that specific test case, that is the main reason to keep the clean process at the end of that unit test, and not for all the test suite cycle. In any case, if I need to move it to the `beforeEach` let me know...